### ISSUE TITLE
Build on save not working

### ISSUE THREAD

**Issue by pylaligand** [concon121/atom-maven#72]
I recently installed the plugin. It recognized my project, configured its classpath, and built it once. Then I tried changing some sources but no build was triggered on save.

If I disable then re-enable the plugin a build gets triggered once but nothing happens on saves again.

If I modify my pom.xml then a new build gets triggered.

No message appears in the console past the lone build.

Atom version: 1.13.1
atom-maven version: 1.1.0

**Comment by concon121** [comment]
Would it be possible to attach a screen shot / paste of the atom console logs and the developer console logs, so I can get a better understanding of what its doing in your environment?

Also, the below is worth mentioning but might not be relevant.  Apologies in advance.  

RE: "Then I tried changing some sources but no build was triggered on save.".  Does this mean you're changing your java code or the pom.xml.  Atom-Maven only rebuilds your project when you modify your pom.xml.

If you want to recompile your java sources on a save (like in eclipse), take a look at the linter-javac plugin I've linked in the readme.  Atom-Maven generates the .classpath file linter-javac needs to do its job, and the .classpath file will only change if your pom file changes!
